run_scans.py

```diff
@@ -134,12 +157,15 @@ def launchTestsArgs(tool, slot, link, chamber,vt1=None,vt2=0,perchannel=False,tr
                          [options.slot for x in range(len(chamber_config))],
                          chamber_config.keys(),
                          chamber_config.values(),
-                         [options.vt1 for x in range(len(chamber_config))],
-                         [options.vt2 for x in range(len(chamber_config))],
+                         [options.scanmin for x in range(len(chamber_config))],
+                         [options.scanmax for x in range(len(chamber_config))],
+                         [options.nevts   for x in range(len(chamber_config))],
+                         [options.vt1     for x in range(len(chamber_config))],
+                         [options.vt2     for x in range(len(chamber_config))],
                          [options.perchannel for x in range(len(chamber_config))],
                          [options.trkdata for x in range(len(chamber_config))],
-                         [options.ztrim for x in range(len(chamber_config))],
-                         [options.config for x in range(len(chamber_config))]
+                         [options.ztrim   for x in range(len(chamber_config))],
+                         [options.config  for x in range(len(chamber_config))],
```

Comment:
I figured you might care because it looks like are you really did here was clean up spacing.